### PR TITLE
Update test targets to net5

### PR DESCRIPTION
**Pull request by WilliamBZA**

### CODE REVIEW

File: src/.editorconfig [hunk 1]
```diff
@@ -151,11 +151,11 @@ dotnet_diagnostic.IDE0020.severity = error
...
 
-dotnet_diagnostic.IDE0083.severity = error
-csharp_style_prefer_not_pattern = true:error
+dotnet_diagnostic.IDE0083.severity = suggestion
+csharp_style_prefer_not_pattern = true:suggestion
 
-dotnet_diagnostic.IDE0078.severity = error
-csharp_style_prefer_pattern_matching = true:error
+dotnet_diagnostic.IDE0078.severity = suggestion
+csharp_style_prefer_pattern_matching = true:suggestion
```
Comment (WilliamBZA) [inline src/.editorconfig:158]: This needs to be done to the repostandards repo

**Comment by andreasohlund** [comment]
Fails on linux with 

>  Skipped Should_for_our_code_exclude_everything_but_NServiceBus_by_default [< 1 ms]
Error: EnsureNoDocumentationIsEmpty: System.UriFormatException : Invalid URI: The hostname could not be parsed.
  Failed EnsureNoDocumentationIsEmpty [16 ms]
  Error Message:
   System.UriFormatException : Invalid URI: The hostname could not be parsed.
  Stack Trace:
     at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at System.Uri..ctor(String uriString)
   at System.UriBuilder..ctor(String uri)
   at NServiceBus.Core.Tests.DocumentationTests.EnsureNoDocumentationIsEmpty() in

**Comment by WilliamBZA** [comment]
Tests are passing now @andreasohlund @dvdstelt

**Review by andreasohlund — APPROVED** [review]
What did you do to fix the Linux tests?

**Comment by WilliamBZA** [comment]
@andreasohlund [switched to](https://github.com/Particular/NServiceBus/pull/6059/files#diff-ceef759e02faaf460ef83e71d47515a5b6d5bb6bed5b6ebcc48a6833e89946adR22) `AppContext.BaseDirectory` according to https://www.hanselman.com/blog/how-do-i-find-which-directory-my-net-core-console-application-was-started-in-or-is-running-from

**Pull request merged**